spec/support/cohort.spec.js

```diff
@@ -0,0 +1,81 @@
+// eslint-disable-next-line import/no-duplicates
```

Comment:
Why are you suppressing this eslint rule?

Reply:
was having some error about it , so this was a quick fix